src/borg/archiver.py

```diff
@@ -2202,11 +2205,22 @@ def _setup_implied_logging(self, args):
             if args.get(option, False):
                 logging.getLogger(logger_name).setLevel('INFO')
 
+    def _setup_topic_debugging(self, args):
+        """Turn on DEBUG level logging for specified --debug-topics."""
+        debug_topics = {
+            'file-compression': 'borg.debug.file-compression',
+        }
```

Comment:
Since this also doubles as an easy way to control just any logger(1) I'd say no extra check is needed.

(1) `debug_topics.get(topic, topic)` -> not in the mapping, just use it directly.